database.go

```diff
@@ -88,19 +89,24 @@ func searchById(id uuid.UUID) (Book, error) {
 }
 
 /* Stores the book into the database, checks and returns it if succeed. */
-func storeOnDB(newBook Book) (fail bool, storedBook Book) {
+func storeOnDB(newBook Book) (Book, error) {
 	sqlStatement := `
 	INSERT INTO bookstable (id, name, price, inventory)
 	VALUES ($1, $2, $3, $4)
 	RETURNING *`
-	createdRow := dbObject.QueryRow(sqlStatement, newBook.ID, newBook.Name, *newBook.Price, *newBook.Inventory)
+	createdRow := dbObjectGlobal.QueryRow(sqlStatement, newBook.ID, newBook.Name, *newBook.Price, *newBook.Inventory)
 	var bookToReturn Book
-	switch err := createdRow.Scan(&bookToReturn.ID, &bookToReturn.Name, &bookToReturn.Price, &bookToReturn.Inventory); err {
-	case sql.ErrNoRows:
-		return true, Book{}
-	case nil:
-		return false, bookToReturn
-	default:
-		panic(err)
+	err := createdRow.Scan(&bookToReturn.ID, &bookToReturn.Name, &bookToReturn.Price, &bookToReturn.Inventory)
+	if err != nil {
+		switch err {
+		case sql.ErrNoRows:
+			{
+				return Book{}, fmt.Errorf("storing on db: %w", errBookNotFound)
+			}
```

Comment:
This should be working fine, but there is not need for the extra brackets.
```suggestion
			return Book{}, fmt.Errorf("storing on db: %w", errBookNotFound)